monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -596,6 +604,7 @@
         },
         {
           alert: 'CephPoolBackfillFull',
+          'for': '1m',
```

Comment:
This is a problem which is not going to dissapear "automagically" or flappling. I think that If this happens we need to warn the user immediately. Why do you want to wait for a minute?